src/Freya.Core/Builder.fs

```diff
@@ -73,6 +73,6 @@ type FreyaBuilder () =
             this.While (enum.MoveNext, this.Delay (fun () -> 
                 body enum.Current)))
 
-/// The instance of <see cref="FreyaBuilder" /> used for constructing
-/// <see cref="Freya{T}" /> computations.
-let freya = new FreyaBuilder ()
+/// The instance of <see cref="CoreBuilder" /> used for constructing
+/// <see cref="Core{T}" /> computations.
+let core = new CoreBuilder ()
```

Comment:
Yikes! I would not like this. Question: could we rename all these things to `OWIN` (or `owin` and `Owin`)? That would better indicate intent and use than `Core`.